deltas/engine/prefabs/player/player.prefab

```diff
@@ -0,0 +1,5 @@
+{
+  "CurrencyStorage": {
+    "amount": 100
+  }
+}
```

Comment:
Why do we need this in GD, but not in LaS?

Reply:
Because LaS makes use of the default amount in Economy but GooeyDefence needs to override it.

Reply:
That means that as soon as somebody changes the default in Economy (a module that itself should be unaware of the modules using it), this will break the balancing of LaS. That sounds like a bad idea. Please also add the override to LaS, even if it's set to the same value as the (current) default in Economy.

Reply:
I made the changes but I couldn't test this in multiplayer as it is taking over 30 minutes for multiplayer to load.

Reply:
It finally loaded but for some reason there seems to be only a single CurrencyStorageComponent shared among all the players. So any item purchased by one player causes a reduction in other players wallet too. I'll try and fix this but it is taking a lot of time for multiplayer to load currently.